openassessment/data.py

```diff
@@ -498,6 +498,56 @@ def collect_ora2_data(cls, course_id):
         ]
         return header, rows
 
+    @classmethod
+    def collect_ora2_summary(cls, course_id):
+        """
+        Query database for aggregated ora2 summary data.
+
+        Args:
+            course_id (string) - the course id of the course whose data we would like to return
+
+        Returns:
+            A tuple containing two lists: headers and data.
+
+            headers is a list containing strings corresponding to the column headers of the data.
+            data is a list of lists, where each sub-list corresponds to a row in the table of all the data
+                for this course.
+
+        """
+
+        items = AssessmentWorkflow.objects.filter(course_id=course_id)
+
+        rows = []
+        for aw in items:
+            statuses = aw.status_details()
+            submission_dict = sub_api.get_submission_and_student(aw.submission_uuid)
+
+            row = [
+                aw.item_id,
+                submission_dict['student_item']['student_id'],
+            ] + [
+                'complete' if statuses.get(step) and statuses[step]['complete'] else '' for step in AssessmentWorkflow.STEPS
+            ] + [
+                1 if aw.staff_score_exists() else 0,
+                1 if aw.status == AssessmentWorkflow.STATUS.done else 0,
+                aw.score if aw.score is not None else '',
+                1 if aw.is_cancelled else 0,
```

Comment:
I'm on the fence about whether all the `if...else` statements should be moved to proper blocks..  but maybe if you just moved them to named variables, so it's clear which one is which?

e.g.
```python
is_staff_grade_received = 1 if aw.staff_score_exists() else 0
is_final_grade_received = 1 if aw.status == AssessmentWorkflow.STATUS.done else 0
```